DOCS/request-new-license.md

```diff
@@ -3,28 +3,29 @@ The SPDX Legal Team appreciates proposals for new free and open source licenses
 
 # How to request a new license or exception be added to the SPDX License List
 
-1. Join the [legal team mailing list](https://lists.spdx.org/g/spdx-legal).
+1.  Review the [license inclusion principles](license-inclusion-principles.md). Please refrain from submitting licenses that clearly do not meet these principles, for example, a license for which the text is not finalized or is only used on your personal project.
 
-2.  Review the [license inclusion principles](license-inclusion-principles.md).
+    1.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure your license or exception has not been previously requested.
 
-3.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested.
-
-4. If your license is not already SPDX License List, submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](license-fields.md).
+2. If your license is not already SPDX License List, submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](license-fields.md).
```

Comment:
typo in the original, copied here: "not already SPDX License List" => "not already **on the** SPDX License List"